apps/frontend/src/App.tsx

```diff
@@ -0,0 +1,7 @@
+import "./App.css";
+
+function App() {
+  return <h1>Martletplace</h1>;
```

Comment:
MartletPlace pls, sorry to be nitpicky

Reply:
My assumption is that wouldn't be capitalized...no?
![image](https://github.com/UVicMartletplace/martletplace/assets/73045936/056d5ef2-271c-472e-84ba-87b52e90e6c3)